native/src/routes/PDFViewModal.tsx

```diff
@@ -40,9 +45,9 @@ const PDFViewModal = ({ route, navigation: _navigation }: PropsType): ReactEleme
         }}
         trustAllCerts={false}
         onError={() => setError(true)}
+        onPressLink={uri => openExternalUrl(uri).catch(() => showSnackbar(t('linkError')))}
```

Comment:
We use `url` for naming everywhere. Even though this may not be completely correct, I would prefer that here.
```suggestion
        onPressLink={url => openExternalUrl(url).catch(() => showSnackbar(t('urlError')))}
```